src/components/Marker/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { FunctionComponent } from 'react';
 import { ChildComponentProps } from 'google-map-react';
-import './styles.scss';
+import './styles.css';
 import Pin from './pin.svg';
```

Comment:
I don't know why it is in the code, but there are some warnings. Did you pull any other branch? I think for now it would be a good idea not to include changes in other files in this commit. I don't know if it is the right approach, but it would get rid of the multiple warnings.

Reply:
I wanted to change all the `SCSS` references to `CSS` since we will be using `CSS` instead for styling components.

Those warnings can mostly be ignored since one is just a console.log on server-side to show the server is on, and the other is due to us not implementing the contentful routes yet (warning will be gone after that issue is complete).

I will go ahead and remove that unused import and suppress those two other warnings.